src/app/clusters/mode-base-server/mode-base-server.cpp

```diff
@@ -193,7 +193,7 @@ CHIP_ERROR Instance::Init()
             }
         }
     }
-
+#ifdef EMBER_AF_PLUGIN_ON_OFF_SERVER
```

Comment:
This file is not including `app/util/config.h`, which is where that define would come from.  Please make sure to include it.